other/doomsday.py

```diff
@@ -46,7 +46,7 @@ def get_week_day(year: int, month: int, day: int) -> str:
     ) % 7
     day_anchor = (
         DOOMSDAY_NOT_LEAP[month - 1]
-        if (year % 4 != 0) or (centurian == 0 and (year % 400) == 0)
+        if (year % 4 != 0) or ((centurian == 0) and (year % 400 != 0))
```

Comment:
```suggestion
        if year % 4 != 0 or (centurian == 0 and year % 400 != 0)
```
I believe some of the parentheses are unnecessary, order of operations should still be clear due to the higher precedence of `%`, `==`, and `!=`.

Reply:
You're right! The expression looks much better without redundant parentheses 👍